src/components/tag/MeeteamTag.tsx

```diff
@@ -88,42 +100,36 @@ const MeeteamTag = ({ tags }: IMeeteamTag) => {
 				<div className='tag__box' onClick={onClickInput}>
 					{tagList.map((tagItem, index) => {
 						return (
-							<div className='tag__item' key={index}>
+							<div className='tag__item' key={tagItem}>
 								<span>{tagItem}</span>
-								<button type='button' onClick={deleteTagItem} id={'' + index}>
-									X
+								<button type='button' onClick={() => deleteTagItem(tagItem)}>
+									<img src={XBtn} />
 								</button>
 							</div>
 						);
 					})}
 					<input
 						type='text'
-						placeholder={
-							isTouched
-								? isDropdownVisible
-									? tagList.length < 20
-										? '태그를 입력하고 엔터를 누르세요.'
-										: '태그는 20개까지 선택할 수 있습니다.'
-									: ''
-								: '태그를 입력하고 엔터를 누르세요.'
-						}
+						placeholder={isTouched ? '' : '태그를 선택하거나 입력해주세요.'}
 						tabIndex={2}
 						disabled={tagList.length < 20 ? false : true}
 						onChange={event => setTagItem(event.target.value)}
 						value={tagItem}
 						onKeyPress={onKeyPress}
 					/>
+					{tagList.length === 0 && <img src={Search} className='icon-search' />}
 					{isDropdownVisible && (
 						<div className='tag-dropdown'>
-							{options.map((tag, index) => (
-								<div
-									className='tag__item option'
-									key={index}
-									onClick={() => onClickTagOptions(tag)}
-								>
-									{tag}
-								</div>
-							))}
+							{!isLoading &&
+								data.map((tag: any) => (
```

Comment:
```suggestion
							{!isLoading &&
								data.map((tag:  {id: string, name: string}) => (
```

any 타입쓰면 린트때문에 빌드 에러 날 것 같아서 해당 타입으로 바꾸는게 좋을 것 같습니다!

Reply:
수정했습니다!!